src/syscall/js/js.go

```diff
@@ -306,6 +308,7 @@ func (v Value) Set(p string, x interface{}) {
 	runtime.KeepAlive(xv)
 }
 
+//go:noescape
```

Comment:
Justification: this will affect whether strings with dynamic contents but statically known length to not escape anymore.